package.json

```diff
@@ -0,0 +1,27 @@
+{
```

Comment:
there should have "start": "node server.js" in the scripts section.
you can do a npm init again to get the proper variables